include/KeyboardShortcuts.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * KeyboardShortcuts.h - Cross-platform handling of keyboard modifier keys
+ *
+ * Copyright (c) 2025 Michael Gregorius
+ * Copyright (c) 2015 Tres Finocchiaro <tres.finocchiaro/at/gmail.com>
```

Comment:
```suggestion
 * Copyright (c) 2025 Tres Finocchiaro <tres.finocchiaro/at/gmail.com>
```
Hello, I'm from the future